pkg/node/state_fsm/fsm.go

```diff
@@ -95,6 +98,11 @@ const (
 	MicroBlockInvEvent = "MicroBlockInv"
 	TransactionEvent   = "Transaction"
 	HaltEvent          = "Halt"
+
+	StopSyncEvent       = "StopSync"
+	StopMiningEvent     = "StopMining"
+	StartMiningEvent    = "StartMining"
+	ChangeSyncPeerEvent = "ChangeSyncPeer"
```

Comment:
The same issue as [this](https://github.com/wavesplatform/gowaves/pull/1107#discussion_r1263171007) one

Reply:
Added TODO for later consideration. I think those `Trigger`-s of FSM can be messages  that comes from channels directly.